spec/models/sorn_spec.rb

```diff
@@ -54,4 +54,68 @@
       expect(sorn.history).to eq "[\"N/A.\"]"
     end
   end
-end
```

Comment:
Here are the specs for .get_action_type. Add example actions to the array here to test any new changes to your code.